redis_bitset.go

```diff
@@ -45,3 +50,25 @@ func (r *RedisBitSet) Test(offsets []uint) (bool, error) {
 
 	return true, nil
 }
+
+func (r *RedisBitSet) Delete() error {
+	n := uint(0)
+	keys := make([]string, 0)
+	for n <= r.maxN {
+		key := fmt.Sprintf("%s:%d", r.keyPrefix, n)
+		keys = append(keys, key)
+		n = n + 1
+	}
+	_, err := r.conn.Do("DEL", strings.Join(keys, " "))
+	return err
+}
```

Comment:
But there's no way for the user to set the max value (aside from setting bits).

Use case:

Start program
Create a bitset, write the nth bit (for a very large n)
Terminate program
Load bitset, immediately call delete -- that'll just attempt to delete the 0th key (and if that prefix is reused, the undeleted keys will be "tainted" with false bits).